src/rest/rest-router.ts

```diff
@@ -50,4 +51,6 @@ subRouter.use(GitHubTokenHandler);
 
 subRouter.use("/org", GitHubOrgsRouter);
 
+subRouter.delete("/subscription/:installationId", SubscriptionDeleteRoute);
```

Comment:
I'd use a `subRouter.use` to be consistent with others. The routes can be `/subscription/:subscriptionId`

Reply:
Updated